cmd/noms/noms.go

```diff
@@ -31,6 +33,13 @@ var commands = []*util.Command{
 	nomsVersion,
 }
 
+type kCommandHandler func(input string) (exitCode int)
```

Comment:
How about `command` and `commandHandler`. The `k` prefix is frequently used as a nemonic to mean 'constant', so it's confusing to see it here.